cbor-schema/plt.cddl

```diff
@@ -254,12 +268,11 @@ token-add-deny-list-event = token-list-update-details
 ; This is a token kernel event, and indicates the account was removed from the deny list.
 token-remove-deny-list-event = token-list-update-details
 
-; The details of a token "pause" event, indicating whether the execution of the "mint", "burn",
-; and "transfer" token operations has been paused.
-token-pause-event = {
-    ; Whether the execution of token operations has been paused.
-    "paused": bool
-}
+; The details of a token "pause" event, indicating whether the execution token operations has been paused.
+token-pause-event = token-pause-state
+
+; The details of a token "unpause" event, indicating whether the execution token operations has been paused.
+token-unpause-event = token-pause-state
```

Comment:
I don't think there should be state included here. Just the fact that pause or unpause came into effect.